src/content/components/inline-loading/style.md

```diff
@@ -0,0 +1,34 @@
+## Color
+
+| Attribute              | SCSS          | HEX      |
+|--------------------|---------------|----------|
+| Spinner      | $ui-05        | #5a6872 |
```

Comment:
We've changed this on the website now to reflect the actual CSS class names. We haven't deployed it yet, but you can take a look at one of the other style pages on the master branch to see what I mean. Do you mind updating?

Reply:
You can see an example here https://github.com/IBM/design-system-website/blob/master/src/content/components/button/style.md

Reply:
Updated the tables, let me know if it's correct or not 😄